src/huggingface_hub/hf_api.py

```diff
@@ -286,6 +305,58 @@ def __str__(self):
         return r
 
 
+class SpaceInfo:
+    """
+    Info about a Space accessible from huggingface.co
+
+    This is a "dataclass" like container that just sets on itself any attribute
+    passed by the server.
+
+    Args:
+        id (`str`, *optional*):
+            id of space
+        sha (`str`, *optional*):
+            repo sha at this particular revision
+        lastModified (`str`, *optional*):
+            date of last commit to repo
+        siblings (`List[Dict]`, *optional*):
+            list of files that constitute the Space
+        private (`bool`, *optional*):
+            is the repo private
+        author (`str`, *optional*):
+            repo author
+    """
+
+    @_deprecate_positional_args
```

Comment:
we've released, so the default is now too early, WDYT?

```suggestion
    @_deprecate_positional_args(version="0.8")
```